src/angleaxis_types.jl

```diff
@@ -172,24 +172,24 @@ function (::Type{RV})(aa::AngleAxis) where RV <: RotationVec
     return RV(aa.theta * aa.axis_x, aa.theta * aa.axis_y, aa.theta * aa.axis_z)
 end
 
-function (::Type{Q})(rv::RotationVec) where Q <: QuatRotation
-    return QuatRotation(AngleAxis(rv))
-end
-
-(::Type{RV})(q::QuatRotation) where {RV <: RotationVec} = RV(AngleAxis(q))
-
-function Base.:*(rv::RotationVec{T1}, v::StaticVector{3, T2}) where {T1,T2}
+@inline function (::Type{Q})(rv::RotationVec) where Q <: QuatRotation
     theta = rotation_angle(rv)
-    if (theta > eps(T1)) # use eps here because we have the 1st order series expansion defined
-        return AngleAxis(rv) * v
+    if theta < sqrt(eps(typeof(theta)))
+        ϕ = theta / (2π)
```

Comment:
```suggestion
        ϕ = theta / π / 2
```
This change fixes https://github.com/JuliaGeometry/Rotations.jl/pull/272#issuecomment-1772759815.